<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="IT3ANCVR" name="app.pages.PageObject" tests="3" failures="0" timestamp="16 Dec 2020 10:56:08 GMT" time="0.073" errors="3">
  <testcase name="clearDriver" time="0.006" classname="app.pages.PageObject">
    <error type="org.openqa.selenium.NoSuchSessionException" message="A session is either terminated or not started
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;IT3ANCVR&#039;, ip: &#039;10.144.242.125&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_161&#039;
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities {appActivity: vodafone.vis.engezly.ui.scr..., appPackage: com.emeint.android.myservices, databaseEnabled: false, desired: {appActivity: vodafone.vis.engezly.ui.scr..., appPackage: com.emeint.android.myservices, deviceName: Google Emulator, fullReset: false, language: ar, locale: EG, noReset: false, platformName: android, systemPort: 10000}, deviceManufacturer: Google, deviceModel: Android SDK built for x86, deviceName: emulator-5554, deviceScreenSize: 1080x1920, deviceUDID: emulator-5554, fullReset: false, javascriptEnabled: true, language: ar, locale: EG, locationContextEnabled: false, networkConnectionEnabled: true, noReset: false, platform: LINUX, platformName: Android, platformVersion: 9, systemPort: 10000, takesScreenshot: true, warnings: {}, webStorageEnabled: false}
Session ID: 583587db-a21d-4aa6-b85f-280d2e014de2">
      <![CDATA[org.openqa.selenium.NoSuchSessionException: A session is either terminated or not started
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: io.appium.java_client.android.AndroidDriver
Capabilities {appActivity: vodafone.vis.engezly.ui.scr..., appPackage: com.emeint.android.myservices, databaseEnabled: false, desired: {appActivity: vodafone.vis.engezly.ui.scr..., appPackage: com.emeint.android.myservices, deviceName: Google Emulator, fullReset: false, language: ar, locale: EG, noReset: false, platformName: android, systemPort: 10000}, deviceManufacturer: Google, deviceModel: Android SDK built for x86, deviceName: emulator-5554, deviceScreenSize: 1080x1920, deviceUDID: emulator-5554, fullReset: false, javascriptEnabled: true, language: ar, locale: EG, locationContextEnabled: false, networkConnectionEnabled: true, noReset: false, platform: LINUX, platformName: Android, platformVersion: 9, systemPort: 10000, takesScreenshot: true, warnings: {}, webStorageEnabled: false}
Session ID: 583587db-a21d-4aa6-b85f-280d2e014de2
at sun.reflect.GeneratedConstructorAccessor24.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:46)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.android.AndroidDriver.execute(AndroidDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:452)
at utilities.TestBase.clearDriver(TestBase.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- clearDriver -->
  <testcase name="appInit" time="0.035" classname="app.pages.PageObject">
    <error type="org.openqa.selenium.SessionNotCreatedException" message="Unable to create a new remote session. Please check the server log for more details. Original error: &#039;wdaLocalPort&#039; must be of type number; &#039;webkitDebugProxyPort&#039; must be of type number
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;IT3ANCVR&#039;, ip: &#039;10.144.242.125&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_161&#039;
Driver info: driver.version: IOSDriver
remote stacktrace: InvalidArgumentError: &#039;wdaLocalPort&#039; must be of type number; &#039;webkitDebugProxyPort&#039; must be of type number
    at validateCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:52:11)
    at parseCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:154:20)
    at processCapabilities (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:197:43)
    at XCUITestDriver.callee$0$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\commands\session.js:36:14)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (&lt;anonymous&gt;)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.callee$0$0 [as createSession] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\build\lib\basedriver\commands\session.js:35:30)
    at XCUITestDriver.createSession$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\lib\driver.js:101:21)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (&lt;anonymous&gt;)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.createSession (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\build\lib\driver.js:220:34)
    at AppiumDriver.callee$2$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\lib\appium.js:244:43)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;IT3ANCVR&#039;, ip: &#039;10.144.242.125&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_161&#039;
Driver info: driver.version: IOSDriver">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
remote stacktrace: InvalidArgumentError: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
    at validateCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:52:11)
    at parseCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:154:20)
    at processCapabilities (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:197:43)
    at XCUITestDriver.callee$0$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\commands\session.js:36:14)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.callee$0$0 [as createSession] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\build\lib\basedriver\commands\session.js:35:30)
    at XCUITestDriver.createSession$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\lib\driver.js:101:21)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.createSession (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\build\lib\driver.js:220:34)
    at AppiumDriver.callee$2$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\lib\appium.js:244:43)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:38)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:84)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:94)
at io.appium.java_client.ios.IOSDriver.<init>(IOSDriver.java:95)
at utilities.TestBase.appInit(TestBase.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
... 37 more
Caused by: org.openqa.selenium.InvalidArgumentException: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
remote stacktrace: InvalidArgumentError: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
    at validateCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:52:11)
    at parseCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:154:20)
    at processCapabilities (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:197:43)
    at XCUITestDriver.callee$0$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\commands\session.js:36:14)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.callee$0$0 [as createSession] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\build\lib\basedriver\commands\session.js:35:30)
    at XCUITestDriver.createSession$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\lib\driver.js:101:21)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.createSession (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\build\lib\driver.js:220:34)
    at AppiumDriver.callee$2$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\lib\appium.js:244:43)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 42 more
]]>
    </error>
  </testcase> <!-- appInit -->
  <testcase name="appInit" time="0.032" classname="app.pages.PageObject">
    <error type="org.openqa.selenium.SessionNotCreatedException" message="Unable to create a new remote session. Please check the server log for more details. Original error: &#039;wdaLocalPort&#039; must be of type number; &#039;webkitDebugProxyPort&#039; must be of type number
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;IT3ANCVR&#039;, ip: &#039;10.144.242.125&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_161&#039;
Driver info: driver.version: IOSDriver
remote stacktrace: InvalidArgumentError: &#039;wdaLocalPort&#039; must be of type number; &#039;webkitDebugProxyPort&#039; must be of type number
    at validateCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:52:11)
    at parseCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:154:20)
    at processCapabilities (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:197:43)
    at XCUITestDriver.callee$0$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\commands\session.js:36:14)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (&lt;anonymous&gt;)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.callee$0$0 [as createSession] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\build\lib\basedriver\commands\session.js:35:30)
    at XCUITestDriver.createSession$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\lib\driver.js:101:21)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (&lt;anonymous&gt;)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.createSession (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\build\lib\driver.js:220:34)
    at AppiumDriver.callee$2$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\lib\appium.js:244:43)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at &lt;anonymous&gt;
    at process._tickCallback (internal/process/next_tick.js:188:7)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;IT3ANCVR&#039;, ip: &#039;10.144.242.125&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_161&#039;
Driver info: driver.version: IOSDriver">
      <![CDATA[org.openqa.selenium.SessionNotCreatedException: Unable to create a new remote session. Please check the server log for more details. Original error: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
remote stacktrace: InvalidArgumentError: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
    at validateCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:52:11)
    at parseCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:154:20)
    at processCapabilities (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:197:43)
    at XCUITestDriver.callee$0$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\commands\session.js:36:14)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.callee$0$0 [as createSession] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\build\lib\basedriver\commands\session.js:35:30)
    at XCUITestDriver.createSession$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\lib\driver.js:101:21)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.createSession (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\build\lib\driver.js:220:34)
    at AppiumDriver.callee$2$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\lib\appium.js:244:43)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:208)
at io.appium.java_client.remote.AppiumCommandExecutor.createSession(AppiumCommandExecutor.java:217)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:239)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
at io.appium.java_client.DefaultGenericMobileDriver.<init>(DefaultGenericMobileDriver.java:38)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:84)
at io.appium.java_client.AppiumDriver.<init>(AppiumDriver.java:94)
at io.appium.java_client.ios.IOSDriver.<init>(IOSDriver.java:95)
at utilities.TestBase.appInit(TestBase.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.reflect.InvocationTargetException
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at io.appium.java_client.remote.AppiumCommandExecutor$1.createSession(AppiumCommandExecutor.java:186)
... 37 more
Caused by: org.openqa.selenium.InvalidArgumentException: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'IT3ANCVR', ip: '10.144.242.125', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_161'
Driver info: driver.version: IOSDriver
remote stacktrace: InvalidArgumentError: 'wdaLocalPort' must be of type number; 'webkitDebugProxyPort' must be of type number
    at validateCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:52:11)
    at parseCaps (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:154:20)
    at processCapabilities (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\capabilities.js:197:43)
    at XCUITestDriver.callee$0$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\basedriver\commands\session.js:36:14)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.callee$0$0 [as createSession] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-base-driver\build\lib\basedriver\commands\session.js:35:30)
    at XCUITestDriver.createSession$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\lib\driver.js:101:21)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at enqueueResult (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:185:17)
    at new Promise (<anonymous>)
    at new F (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\core-js\library\modules\$.export.js:30:36)
    at AsyncIterator.enqueue (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:184:12)
    at AsyncIterator.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at Object.runtime.async (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:209:12)
    at XCUITestDriver.createSession (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\appium-xcuitest-driver\build\lib\driver.js:220:34)
    at AppiumDriver.callee$2$0$ (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\lib\appium.js:244:43)
    at tryCatch (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:67:40)
    at GeneratorFunctionPrototype.invoke [as _invoke] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:315:22)
    at GeneratorFunctionPrototype.prototype.(anonymous function) [as next] (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:100:21)
    at GeneratorFunctionPrototype.invoke (C:\Users\V19MOsman\AppData\Local\appium-desktop\app-1.6.2\resources\app\node_modules\appium\node_modules\babel-runtime\regenerator\runtime.js:136:37)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.W3CHandshakeResponse.lambda$errorHandler$0(W3CHandshakeResponse.java:62)
at org.openqa.selenium.remote.HandshakeResponse.lambda$getResponseFunction$0(HandshakeResponse.java:30)
at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:126)
at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)
at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)
at java.util.stream.AbstractPipeline.copyInto(Unknown Source)
at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)
at java.util.stream.AbstractPipeline.evaluate(Unknown Source)
at java.util.stream.ReferencePipeline.findFirst(Unknown Source)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:128)
... 42 more
]]>
    </error>
  </testcase> <!-- appInit -->
</testsuite> <!-- app.pages.PageObject -->
